pkg/controller/core/workload_controller.go

```diff
@@ -179,6 +168,17 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return r.reconcileNotReadyTimeout(ctx, req, &wl)
 	}
 
+	// If the workload has rejected admission checks mark it as finished.
```

Comment:
Maybe remind us what is the state here: the workload is admitted.

Reply:
done